src/main/java/frc/robot/commands/DefaultAimCommand.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.AimingConstants.AimState;
+import frc.robot.constants.AimingConstants;
+import frc.robot.constants.JoystickConstants;
+import frc.robot.subsystems.AimingSubsystem;
+import frc.robot.subsystems.Input;
+
+public class DefaultAimCommand extends Command {
+
+  private final AimingSubsystem mAimingSubsystem;
+
+  //TODO: Implement Input Using an Xbox Controller(You can choose the button mapping)
```

Comment:
Can you create an issue to track this work, and paste the link to the issue here in the comment?

Reply:
We resolved the issue and added input buttons.